### PR TITLE
(TK-251) Convert symbol keys to strings

### PR DESCRIPTION
**Pull request by fpringvaldsen**
When parsing a Hash in ConfigValueFactory, automatically convert
all symbol keys to strings.

### CODE REVIEW

**Comment by fpringvaldsen** [comment]
This PR addresses https://github.com/puppetlabs/ruby-hocon/issues/60

File: spec/unit/typesafe/config/config_value_factory_spec.rb [hunk 1]
```diff
@@ -56,6 +57,23 @@
       expect(value.unwrapped).to eq(map)
     end
 
+    it "should convert symbol keys in a map to string keys" do
+      orig_map = {a: 1, b: 2}
```
Comment (cprice404) [inline spec/unit/typesafe/config/config_value_factory_spec.rb]: @fpringvaldsen what about nested maps?

**Comment by fpringvaldsen** [comment]
@cprice404 Nested maps will now be pre-processed as well.

**Comment by jpinsonault** [comment]
Looks good to me :+1:

**Pull request merged**